library/std/src/sync/barrier.rs

```diff
@@ -68,6 +68,8 @@ impl Barrier {
     /// A barrier will block `n`-1 threads which call [`wait()`] and then wake
     /// up all threads at once when the `n`th thread calls [`wait()`].
     ///
+    /// If `n` is 0, the barrier acts the same as `n` is 1.
+    ///
```

Comment:
Optional suggestion: I think it's a bit nicer to clarify this by the math

 > A barrier will block `n - 1` (saturating) threads which call...

But don't feel too strongly. Either change to that, or update the current version to "acts the same as *when* `n` is 1" then r=me.

Reply:
```suggestion
    /// A barrier will block `n-1` (saturating) threads which call [`wait()`] and then wake
    /// up all threads at once when the `n`th thread calls [`wait()`].
    ///
```